File: atv-1/python/calculadora/main.py
from Calcula import Calcula

calc = Calcula

n1 = int(input('Insira um valor'))
n2 = int(input('Insira outro valor'))
operador = input('Escola a operação(+, -, * ou /)')
print(calc.calc(n1, n2, operador))

File: atv-1/php/calculadora/main.php
<?php
include_once 'Calc.php';

$calc = new Calc;

$n1 = readline('Insira o primeiro valor: ');
$n2 = readline('Insira o segundo valor: ');

$operador = readline('Escolha qual será a operação (+, -, *, /): ');

echo 'o resultado é ', $calc->calcular($n1, $n2, $operador);
